modules/configuring-default-route-timeouts-globally.adoc

```diff
@@ -0,0 +1,104 @@
+
+.Procedure
+
+. Set the value of `ROUTE_HAPROXY_TIMEOUT` in your subscription to your required timeout in seconds:
++
+.Example of setting the `ROUTE_HAPROXY_TIMEOUT` value to 900 seconds
+[source,terminal]
+----
+$ oc patch subscription.operators.coreos.com serverless-operator -n openshift-serverless --type='merge' -p '{"spec": {"config": {"env": [{"name": "ROUTE_HAPROXY_TIMEOUT", "value": "900"}]}}}'
+----
++
+`ROUTE_HAPROXY_TIMEOUT` is an enviroment managed by the Serverless Operator and by default is set to `600`. This subscription update causes pods to be redeployed in the `openshift-serverless` namespace.
+
+[NOTE]
+====
+If you created your routes manually and disabled auto-generation with the annotation `serving.knative.openshift.io/disableRoute`, you can configure the timeouts directly in the route definitions.
+====
+
+. Set the maximum revision timeout as follows in the `KnativeServing` custom resource (CR):
++
+.Example of `max-revision-timeout-seconds` set to 900 seconds
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    defaults:
+      max-revision-timeout-seconds: "900"
+----
++
+The Serverless Operator automatically adjusts the activator's `terminationGracePeriod` to the set maximum revision timeout value to avoid request termination in cases where activator pods are being terminated themselves.
++
+You can verify that the timeout has been set by running the following command: 
++
+[source,terminal]
+----
+$ oc get deployment activator -n knative-serving -o jsonpath="{.spec.template.spec.terminationGracePeriodSeconds}"
+900
+----
+
+. Optional: Adjust the load balancer timeout if necessary for your cloud provider:
++
+.Example of load balancer timeout adjustment for AWS Classic LB 
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default --type=merge --patch='{"spec":{"endpointPublishingStrategy": {"type":"LoadBalancerService", "loadBalancer": {"scope":"External", "providerParameters":{"type":"AWS", "aws": {"type":"Classic", "classicLoadBalancer": {"connectionIdleTimeout":"20m"}}}}}}}'
+----
+
+.  Deploy a Knative service with the desired timeouts less or equal to the maximum set:
++
+.A service definition with timeouts set to 800 seconds
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name:
+spec:
+  template:
+    spec:
+      timeoutSeconds: 800
+      responseStartTimeoutSeconds: 800
+----
++
+[IMPORTANT]
+====
+When using {SMProductShortName}, if the activator pod is stopped while a long-running request is in-flight, the request will be interrupted.
+To avoid request interruptions, you must adjust the `terminationDrainDuration` value in the `ServiceMeshControlPlane`: 
```

Comment:
```suggestion
To avoid request interruptions, you must adjust the `terminationDrainDuration` value in the `ServiceMeshControlPlane` CR: 
```

- Add an object name. See, https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references